rdkit/Chem/UnitTestRandomize.py

```diff
@@ -0,0 +1,51 @@
+#
+#  Copyright (C) 2017 Esben Jannik Bjerrum
+#
+#   @@ All Rights Reserved @@
+#  This file is part of the RDKit.
+#  The contents are covered by the terms of the BSD license
+#  which is included in the file license.txt, found at the root
+#  of the RDKit source tree.
+#
+from __future__ import print_function
+
+import unittest
+
+from rdkit import Chem
+from rdkit.Chem import Randomize
+import random
+
+
+def MolBlockToSmiles(mb):
+  return Chem.MolToSmiles(Chem.MolFromMolBlock(mb))
```

Comment:
It's likely not relevant here, but normally you would want to include the `isomericSmiles=True` argument to `Chem.MolToSmiles()` so that you get stereochemistry.